python/qupled/quantum.py

```diff
@@ -280,3 +280,129 @@ def setGuess(self, fileName : str) -> None:
+                 iterationsAlpha : int = 50,
+                 errorIntegrals : float = 1.0e-5,
+                 threads : int = 1):
+        # Allowed theories
+        self.allowedTheories = ["QVSSTLS"]
+        # Set theory
+        self.inputs : qupled.qupled.QVSStlsInput = qp.QVSStlsInput() #: Inputs to solve the scheme.
+        self._setInputs(coupling, degeneracy, "QVSSTLS", chemicalPotential, cutoff,
+                        error, fixed, mixing, guess, iterations, matsubara,
+                        outputFrequency, recoveryFile, resolution, scheme2DIntegrals,
+                        alpha, couplingResolution, degeneracyResolution, errorAlpha,
+                        iterationsAlpha, errorIntegrals, threads)
+        # Scheme to solve
+        self.scheme : qp.QVSStls = None
+        # File to store output on disk
+        self.hdfFileName = None
+
+        # Setup inputs object
+    def _setInputs(self,
+                   coupling : float,
+                   degeneracy : float,
+                   theory : str,
+                   chemicalPotential : list[float],
+                   cutoff : float,
+                   error : float,
+                   fixed : str,
+                   mixing : float,
+                   guess : qp.QstlsGuess,
+                   iterations : int,
+                   matsubara : int,
+                   outputFrequency : int,
+                   recoveryFile : str,
+                   resolution : float,
+                   scheme2DIntegrals : str,
+                   alpha : list[float],
+                   couplingResolution : float,
+                   degeneracyResolution : float,
+                   errorAlpha : float,
+                   iterationsAlpha : int,
+                   errorIntegrals : float,
+                   threads : int) -> None:
+        super()._setInputs(coupling, degeneracy, theory, chemicalPotential,
+                   cutoff, error, fixed, mixing, guess, iterations, matsubara,
+                   outputFrequency, recoveryFile, resolution, scheme2DIntegrals, threads)
+        self.inputs.alpha = alpha
+        self.inputs.couplingResolution = couplingResolution
+        self.inputs.degeneracyResolution = degeneracyResolution
+        self.inputs.errorAlpha = errorAlpha
+        self.inputs.iterationsAlpha = iterationsAlpha
+        self.inputs.intError = errorIntegrals
+
+    # Compute
+    @qu.MPI.recordTime
+    @qu.MPI.synchronizeRanks
+    def compute(self) -> None:
+        """ Solves the scheme and saves the results to an hdf file. Exactly as it is done by 
+        :func:`qupled.classic.QStls.compute` .The auxiliary density response dataframe can be accessed as `adr`.
+        """
+        
+        self._checkInputs()
+        self.scheme = qp.QVSStls(self.inputs)
+        status = self.scheme.compute()
+        self._checkStatusAndClean(status)
+        self._setHdfFile()
+        self._save()
+
+    @qu.MPI.runOnlyOnRoot
+    def _save(self) -> None:
```

Comment:
This method might not be necessary because it seems to be doing the exact same thing as QStls::_save

Reply:
This method I added because of how multiple inheritance works in Python. Since QVSStls inherits first from qc.VSStls then the _save() method that will be called will be qc.VSStls::_save and not QStls::_save.

Reply:
Ok! Leave it like this then